src/sentry/utils/data_filters.py

```diff
@@ -0,0 +1,48 @@
+"""
+sentry.utils.data_filters.py
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import fnmatch
+
+
+class FilterTypes(object):
+    ERROR_MESSAGES = 'error_messages'
+    RELEASES = 'releases'
+    BLACKLISTED_IPS = 'blacklisted_ips'
```

Comment:
I think I need more of an explanation for this - do you want me to remove this from the enum?